jbmc/src/java_bytecode/java_object_factory_parameters.h

```diff
@@ -25,6 +25,9 @@ struct java_object_factory_parameterst final : public object_factory_parameterst
   /// Force numerical primitive inputs to fall within the interval
   integer_intervalt assume_inputs_interval;
 
+  /// Force double and float inputs to be integral
```

Comment:
:pick: Add `This does not apply to array elements`.

Reply:
I wouldn't propagate the restrictions everywhere, it's in the option documentation.